text/0521-find-by-identifier.md

```diff
@@ -0,0 +1,122 @@
+- Start Date: (fill me in with today's date, 2019-08-29)
+- Relevant Team(s): EmberData
+- RFC PR: https://github.com/emberjs/rfcs/pull/521
+- Tracking: (leave this empty)
+
+# [DATA] findRecord/peekRecord via Identifier
+
+## Summary
+
+Users should be able to peek or find records based on `lid`.
```

Comment:
Can you include a succinct definition of "lid" (either here or below in detailed design)? It seems that saying "identifier" would have made more sense to me when I read through it.

Reply:
we use `identifier` to mean an object with the `lid` property, whereas `lid` is the unique string itself.